### PR TITLE
Newcomers_Guide: Update step 7 note

### PR DESCRIPTION
**Pull request by MihneaVorovenci**
Fixes https://github.com/coala/documentation/issues/183

### CODE REVIEW

File: Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -190,8 +190,8 @@ For more information about reviewing code, check out
 
 .. note::
 
-    Reviewing code helps you by watching other people's mistakes and not making
-    them yourself in the future!
+    Reviewing code helps you improve the quality of your work and gets you one step closer
+to being an efficient developer.
```
Comment (sils) [inline Developers/Newcomers_Guide.rst]: efficient is kind of almost negatively connotated IMO, maybe like `being a better developer`?

Comment (sils) [inline Developers/Newcomers_Guide.rst]: also missing indentation in this line

**Comment by sils** [comment]
please address gitmate comments and rebase(read the docs!!!)

**Comment by Mixih** [comment]
@MihneaVorovenci Status update?

**Comment by Adrianzatreanu** [comment]
closing due to high inactivity

**Pull request closed without merging**